internal/configure.go

```diff
@@ -156,6 +156,21 @@ func getWalFolderPath() string {
 	return DefaultDataFolderPath
 }
 
+func getRelativeWalFolderPath() string {
+
+	dataFolderPath := filepath.Join("pg_wal")
+	if _, err := os.Stat(dataFolderPath); err == nil {
+		return dataFolderPath
+	}
+
+	dataFolderPath = filepath.Join("pg_xlog")
+	if _, err := os.Stat(dataFolderPath); err == nil {
+		return dataFolderPath
+	}
+
+	return DefaultDataFolderPath
```

Comment:
let's use this function in the above getWalFolderPath() too

Reply:
Thanks. Simplified the function call